src/components/buttons/delete-button.js

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 
```

Comment:
Please, put `delete-button.js` inside its own folder `delete-button` and convert it to normal component folder

Reply:
Done!